src/OrderFormAcceptanceTests.TestData/Organisation.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Linq;
+using Bogus;
+using OrderFormAcceptanceTests.TestData.Utils;
+
+namespace OrderFormAcceptanceTests.TestData
+{
+    public sealed class Organisation
+    {
+        public Guid OrganisationId { get; set; }
+        public string Name { get; set; }
+        public string Address { get; set; }
+        public string OdsCode { get; set; }
+        public string PrimaryRoleId { get; set; }
+        public int CatalogueAgreementSigned { get; set; } = 0;
+        public DateTime LastUpdated { get; set; }        
+
+        public Organisation RetrieveRandomOrganisationWithNoUsers(string connectionString)
+        {
+            var query = @"SELECT [dbo].[Organisations].* FROM [dbo].[Organisations]
+                          LEFT JOIN AspNetUsers on AspNetUsers.PrimaryOrganisationId=[Organisations].OrganisationId
+                          WHERE AspNetUsers.Id IS NULL;";
```

Comment:
I'd use table aliases here, so that the query would become

```sql
SELECT o.*
  FROM dbo.Organisations AS o
  LEFT OUTER JOIN dbo.AspNetUsers AS u
       ON u.PrimaryOrganisationId = o.OrganisationId
 WHERE u.Id IS NULL;
```
I find that a bit more readable, although some of the readability is lost when transferred to a c# string literal of course. It could also be written like this

```sql
SELECT *
  FROM dbo.Organisations AS o
 WHERE NOT EXISTS (
       SELECT *
         FROM dbo.AspNetUsers AS u
        WHERE u.PrimaryOrganisationId = o.OrganisationId);
```
I think I'd probably go with the second version as I think the intent is slightly clearer (it often performs better too but I'd say that's a moot point in this context). Some people can find the correlated subquery syntax a bit confusing though.

I'll leave it up to you. Feel free to use either of my suggestions or to not change anything and leave it as is.

Reply:
I like the second query, that's getting added